flush_loader.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#USB=/dev/ttyUSB0
+USB=/dev/ttyUSB1
+BL2_EL3=1
+
+case "$1" in
+"hikey")
+	echo "Download recovery images into HiKey platform"
+	if [ $BL2_EL3 ]; then
+		echo "Flush recovery.bin into eMMC"
+		sudo python hisi-idt.py -d $USB --img1 recovery.bin
+		sudo fastboot flash loader l-loader.bin
+	else
+		echo "Flush l-loader.bin into eMMC"
+		sudo python hisi-idt.py -d $USB --img1 l-loader.bin
+	fi
+	sudo fastboot flash fastboot fip.bin
+	;;
+"hikey960")
+	echo "Download recovery images into HiKey960 platform"
+	if [ $BL2_EL3 ]; then
+		echo "Flush recovery.bin into UFS"
```

Comment:
Same as above

Reply:
When $BL2_EL3 is active, we have recovery.bin.